packages/core/videoClient.ts

```diff
@@ -301,6 +301,65 @@ const getAllTranscriptsAccessLinkFromRoomName = async (roomName: string) => {
   return videoAdapter?.getAllTranscriptsAccessLinkFromRoomName?.(roomName);
 };
 
+const submitBatchProcessorTranscriptionJob = async (recordingId: string) => {
```

Comment:
Non-blocking: Something doesn't sit right with me having these Daily specific functions as a part of the video client. Have you thought of any alternatives of where these could live?

Reply:
Not sure. probably CalVideoClient